netci.groovy

```diff
@@ -154,7 +154,6 @@ class Constants {
                'ilrt',
                'r2r',
                'longgc',
-               'formatting',
```

Comment:
Instead of this, can you just remove the default trigger? (or just wait for https://github.com/dotnet/coreclr/pull/24376)?

Reply:
The change to not install the .NET CLI will break the Jenkins nightly/master runs that don't have a .NET CLI installed, so that's why I removed the trigger.